<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteControl</class>
 <widget class="QWidget" name="RemoteControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <widget class="QToolButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QToolButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QToolButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="escButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QToolButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
